tests/nexus/test_nexus.py

```diff
@@ -146,7 +146,7 @@ def test_get_node_at_nxdl_path():
 
     nxdl_file_path = os.path.join(
         local_dir,
-        "../../pynxtools/definitions/contributed_definitions/NXiv_temp.nxdl.xml",
+        "../../src/pynxtools/definitions/contributed_definitions/NXiv_temp.nxdl.xml",
```

Comment:
Some test data files are in tests/data which are now also copied to src/pynxtools/data/
Should not we just have one place for them and pick from up from there?

Reply:
The reason why I moved them into the repo is that if they are not there, the example reader cannot be instantiated when you pip install pynxtools (since we don't include the tests/data in the build). Thus, with the new src layout, it cannot find it when running the tests because we actually test the repo as it would be installed. However, in the test function defined in tests/, we also load these files. 

I removed these files from the tests/dat and now point the test_functions to src/pynxtools/data/.